app/src/main/java/org/fossasia/susi/ai/MainActivity.java

```diff
@@ -33,16 +45,20 @@
 import butterknife.ButterKnife;
 import butterknife.OnClick;
 
-public class MainActivity extends AppCompatActivity {
 
+public class MainActivity extends AppCompatActivity {
+    private EmojiPopup emojiPopup;
+    EmojiEditText editText;
     @BindView(R.id.coordinator_layout)
     CoordinatorLayout coordinatorLayout;
     @BindView(R.id.rv_chat_feed)
     RecyclerView rvChatFeed;
     @BindView(R.id.iv_image)
     ImageView ivImage;
-    @BindView(R.id.et_message)
-    EditText etMessage;
+    @BindView(R.id.rl)
+    ViewGroup rootView;
```

Comment:
What is r1 here? If you have changed any layout xml files according to this, add them in your next PR if you're going to open one.